src/components/form/form.stories.mdx

```diff
@@ -77,7 +77,31 @@ When either `errorCount` or `warningCount` or both are provided, summary with th
 </Canvas>
 
 <Canvas>
-  <Story name="with both errors and warnings summary" story={stories.WithBothErrorsAndWarningsSummary} />
+  <Story
+    name="with both errors and warnings summary"
+    story={stories.WithBothErrorsAndWarningsSummary}
+  />
+</Canvas>
+
+### Optional and Required Fields
```

Comment:
**suggestion**: as the relevant optional and required props belong to `Textbox` instead of `Form`, I would amend the story description slightly so its a bit clearer, as to me it currently reads that `Form` provides these props which may be confusing.

One way we could amend this is to tweak the description like so:

```mdx
### Example with optional and required fields

<Canvas>
  <Story name="example with optional and required fields" story={/* Story with optional and required fields */>}>
</Canvas>
```

so it just demonstrates how the `Textbox`es would look in a typical form.